.buildkite/build-test-omnibus.sh

```diff
@@ -122,7 +122,7 @@ then
       echo "      - \"-Command\""
       echo "      propagate-environment: true"
       echo "      environment:"
-      echo "        - CHEF_FOUNDATION_VERSION"
+      echo "        - CHEF_FOUNDATION_VERSION=$CHEF_FOUNDATION_VERSION"
```

Comment:
This may not have to change likely

Reply:
It should not have to change, and its required for setting an ENV var and passing it to docker.